### DISCUSSION TITLE
Problems with viewing (some) streaming content

### DISCUSSION THREAD

**Discussion by pexatar** [zen-browser/desktop#1831] [General]
Hello everyone, I hope I'm writing in the right place.

First of all, I would like to compliment you on the great work you're doing. Even though it’s not 'perfect', I’ve decided to use Zen as my default browser. That being said, I’d like to report a small issue I’ve encountered. Specifically, when trying to view content from Netflix, Crunchyroll, and Now TV, error messages appear.

![netflix](https://github.com/user-attachments/assets/7064bfe3-1c4a-4cdc-a770-5e5dae3aaf6d)
Here it says: 
_We apologize for the interruption.
This title is not available for instant streaming. Please try another title. - 1044
Error code E100_
<hr /><br />

![chrunchyroll](https://github.com/user-attachments/assets/17c630fb-53e7-4f89-8bff-c99884e1f76b)
Here it says: 
_Oops! Something went wrong.
Please try again._
<hr /><br />

![nowtv](https://github.com/user-attachments/assets/55dc79e5-67a7-467c-ab42-f22a66afba04)
Here it says: 
_Oh no! This browser is not supported.
Please check our Support for more information on supported browsers.
UNSUPPORTED_BROWSER_
<hr /><br />

It's not a big problem, in these cases I use a different browser. More than anything, I wanted to report this issue to you, which you may already be aware of, in order to further improve your browser.

For completeness, I'm providing the following information:
- I'm using version 1.0.1-a.7 (64 bit) but the problem was also present in previous versions.
- I'm using a Windows 11 operating system with the following characteristics:
    Edition:    Windows 11 Pro
    Version:    23H2
    Build:    22635.4291
    Experience:    Windows Feature Experience Pack 1000.22700.1045.0
- The PC I'm using has the following specifications:
    CPU:    12th Gen Intel(R) Core(TM) i5-12400F   2.50 GHz
    RAM:    32.0 GB (31.8 GB usable)
    GPU: NVIDIA GeForce RTX 4070
    DirectX: DirectX 12

I hope this report can be helpful to you 😊

**Comment by thyeun** [comment]
@pexatar This is because DRM have to be paid, and now zen browser still less funding, and pls kindly read the info regarding zen browser. 

Read here [DRM](https://docs.zen-browser.app/faq)

**Reply by pexatar** [reply]
I'm sorry I didn't read the information about Zen Browser earlier, and thank you for your response 😉

**Comment by wnffldktm** [comment]
It's happened again:
1. Auto logout each time reload Netflix
2. Auto logout just after arrived in Home > TV Shows / Movies / New & Popular etc.
3. Try to watch any random film > Welcomed with Error Code 1044

Happened in:
- [Release build - 1.6b (2025-01-09)](https://github.com/zen-browser/desktop/releases/tag/1.6b)
- [Twilight build - 1.7t (2025-01-12 at 00:07:21)](https://github.com/zen-browser/desktop/releases/tag/twilight)
- MacBook Pro 13-inch 2017, Two Thunderbolt 3 ports
- 2,3 GHz Dual-Core Intel Core i5
- Intel Iris Plus Graphics 640 1536 MB
- 8 GB 2133 MHz LPDDR3
- Ventura 13.7.3

I've tried:
1. On and Off between Play DRM-controlled content and Use hardware acceleration if available > Four scenarios
2. Clear cache and site information
3. Repatching two Zen Browser build-release
4. Knowing that indeed there's huge amount of money to licensed [DRM](https://docs.zen-browser.app/faq#why-cant-zen-browser-play-drm-protected-content)

Never mind, I already typed in here. So, let's wait for Zen Browser had enough funding to get DRM license.
Salute!

**Reply by wnffldktm** [reply]
This way, it affected if you want to use Spotify Desktop as well, and others.

Zen Browser already good!
It let us know which one that contain DRM or no through it's indicator in address bar, it's helpful.

**Comment by lambertjosh** [comment]
Getting this error with Hulu. Error code P-DEV313. The ads play fine, but the actual video content errors out. Works fine on firefox.

**Reply by MrPiggy105** [reply]
Yep, it's because all browsers that want to play DRM protected content (like music and movie streaming, e.g. Netflix or Spotify) need a Widevine license from Google. Other browsers like Firefox have one but Zen does not (even though it's based on Firefox, it doesn't get its license). It requires approval from Google and more funding, so for the time being you will need to find another browser/app for DRM content. Or use Linux.

**Comment by cliff-4** [comment]
As I understand, this is mainly an issue with DRM content and the fact that Zen browser does not have a license. Can anyone explain the exact procedure and requirement for Zen browser to be able to procure one such license? Is funding the major blocker or is there more to it?
